jmeter/Gemfile

```diff
@@ -3,4 +3,4 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.4'
 
-gem 'ruby-jmeter'
+gem 'ruby-jmeter', '>= 3.1.08'
```

Comment:
This was bumped previously by dependabot but in the lock file. I suggest we close this cc @duncanjbrown

Reply:
So dependabot has already brought us to a good enough version yet snyk still raises this PR? Happy to close if so!